pandas/core/accessor.py

```diff
@@ -193,122 +193,119 @@ def __get__(self, obj, cls):
         return accessor_obj
 
 
+@doc(klass="", others="")
```

Comment:
Is this decorated here for a particular reason?

Reply:
We need to replace the variables, otherwise the values in parenthesis would be in the doc (and `doc` also keeps the original template)

Reply:
Yes. This decorated to force a string format function to be applied. 

I also think that this empty decorator may be a little bit miss-leading. One other possibility could be `@doc(klass="klass", others="others")`. I am not sure which one (use empty string or the arguments' name) is better here. Open for any suggestions and advice.